### PR TITLE
2378: complete correct work item when serving court-issued document

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/useCases/editDocketEntry/saveIntermediateDocketEntryInteractor.js [hunk 1]
```diff
@@ -50,17 +50,15 @@ exports.saveIntermediateDocketEntryInteractor = async ({
     documentId: entryMetadata.documentId,
   });
 
-  const workItemsToPutInProgress = currentDocument.workItems
-    .filter(wi => wi.isQC === true)
-    .filter(wi => !wi.inProgress);
+  const workItemToPutInProgress = currentDocument.getQCWorkItem();
```
Comment (kkoskelin) [inline shared/src/business/useCases/editDocketEntry/saveIntermediateDocketEntryInteractor.js:53]: `getQCWorkItem` does not test the `inProgress` attribute.  Is it redundant at this point?

**Review by kkoskelin — APPROVED** [review]
Cool.  Just one minor question.

File: shared/src/business/useCases/editDocketEntry/saveIntermediateDocketEntryInteractor.js (hunk 1 above)
Comment (rachaelparris) [reply shared/src/business/useCases/editDocketEntry/saveIntermediateDocketEntryInteractor.js:53]: I check that value the next line down in an if statement before making changes to the work item.

**Review by wbyoko — COMMENTED** [review]
in `completeDocketEntryQCInteractor.js` I don't know if every document that gets created has a qc work item. Things like 'Notice of Docket Change'. So you may want to make sure that a qc work item exists before you act on it.`

**Comment by rachaelparris** [comment]
> in `completeDocketEntryQCInteractor.js` I don't know if every document that gets created has a qc work item. Things like 'Notice of Docket Change'. So you may want to make sure that a qc work item exists before you act on it.`

Gotcha. Addressed!

**Pull request merged**